patches/chrome-browser-resources-settings-lifetime_browser_proxy.ts.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/chrome/browser/resources/settings/lifetime_browser_proxy.ts b/chrome/browser/resources/settings/lifetime_browser_proxy.ts
+index 88d9caa34016bcf9d87a908303e77bb89df48b81..a2827a32a3885c05da3722ce99328e1033fe9461 100644
+--- a/chrome/browser/resources/settings/lifetime_browser_proxy.ts
++++ b/chrome/browser/resources/settings/lifetime_browser_proxy.ts
+@@ -9,6 +9,13 @@ export interface LifetimeBrowserProxy {
+   // Triggers a browser relaunch.
+   relaunch(): void;
+ 
++  // <if expr="is_macosx">
++  // Use separate api for relaunch after update on Mac.
```

Comment:
these comments should be removed and why are we commenting out is_macosx?

Reply:
It looks to me like we just moved these chunks from one .ts file to another. These we all added in https://github.com/brave/brave-core/pull/1041/files?file-filters%5B%5D=.patch#diff-3e7288be345e538182a9011a060e79ebf1532fb07c6052b074c4e2c74ee7efc0 cc: @simonhong